oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -76,6 +77,22 @@
      */
     String getCommandLine();
 
+    /**
+     * Gets the command-line arguments of the process.
+     *
+     * <p>On Windows, where the OS models command-line arguments as a single string, this method
```

Comment:
I'm not sure what this means!  Make it clear what the end user should expect to see as the tokenization.